lib/blueberry_rails/version.rb

```diff
@@ -1,7 +1,7 @@
 module BlueberryRails
   VERSION = '0.3.3'
 
-  RUBY_VERSION = '2.1.5'
+  RUBY_VERSION = '2.2.1'
 
-  RAILS_VERSION = '4.1.9'
+  RAILS_VERSION = '4.2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.